kiali-server/templates/service.yaml

```
@@ -34,11 +34,17 @@ spec:
   {{- end }}
     protocol: TCP
     port: {{ .Values.server.port }}
+    {{- if eq .Values.deployment.service_type "NodePort" }}
+    nodePort: {{ .Values.server.nodePort }}
```

use snake_case for consistency:
```suggestion
    nodePort: {{ .Values.server.node_port }}
```